test/core_tests.py

```diff
@@ -240,7 +240,7 @@ def test_error_file(self):
             'workspace_id': self.wsID,
             'sample_file': sample_file,
             'file_format': "ENIGMA",
-            'header_row_index': 2,
+            'header_row_index': 6,
```

Comment:
a little confused by this

Reply:
I moved the header down when I modified `test/data/error_file.csv `